addons/mrp/views/stock_move_views.xml

```diff
@@ -29,6 +29,8 @@
                                 <span> / </span>
                                 <span><field name="reserved_availability" nolabel="1"/></span>
                             </div>
+                            <field name="operation_id" attrs="{'invisible': ['|', ('parent.routing_id', '=', False), ('id', '!=', False)]}" groups="mrp.group_mrp_routings"/>
```

Comment:
a readonly  attrs="{'readonly': [('state', '!=', 'draft')]}" isn't needed?

Reply:
Not needed as it's not possible to fill the field in any other status than draft. but can be added to be sure